index.rst

```diff
@@ -10,20 +10,849 @@
+
+In the case where a more complex issue arises (e.g., a 2% increase in bad pixels is observed), this is addressed by the calibration team offsite and is not immediately reported to the summit team.
+When the calibrations used on the summit need to be updated, this is the role of the calibration scientist and is not the responsibility of the observer.
+Furthermore, this cadence is expected to be slow (months) and is therefore outside the scope of this charge.
+
+.. [#] The `Watcher CSC <https://ts-watcher.lsst.io/>`_ is provided a list of "rules" that it ensures the system is always obeying. If a rule is violated, such as a temperature going out of specification, the an alert or alarm is issued to the observer via the LOVE interface. The alarm stays in place until the rule is no longer violated and the original alert has been acknowledged. The Watcher is not able to perform analyses and only evaluates simple conditions.
+
+.. _Deliverable 2:
+
+Deliverable 2: Rapid Analysis Calculated Metrics
+------------------------------------------------
+
+.. note::
+
+   The deliverable description from the `charge`_ has been directly copied here to ease readability.
+
+
+   2. (`SITCOM-180`_, `SITCOM-173`_) Define which metrics, analyses and artifacts must be calculated and on what timescale they must be evaluated and reported to support commissioning/operations.
+
+      This is to evaluate if a "rapid processing" of data is required, what specific calculations are required.
+      This list should include the relevant camera specific calculations (which are currently performed by the EO testing data reduction).
+      This is expected to inform the answer to the next charge task.
+
+
+It is important to note that the charge question above refers to "rapid processing."
+We intentionally avoid the use of this term and have adopted the phrasing, "Rapid Analysis" instead.
+This is to avoid any potential confusion with Prompt Processing, which is a discussed previously in the `Rapid Analysis`_ section.
+
+Numerous calculations are required to evaluate both camera and control system health and performance on rapid timescales.
+The data products discussed in this section are limited to scalars and/or arrays and do *not* include diagnostic plots and/or figures (visualization use cases are discussed separately).
+The large majority of data products needed on rapid timescales are produced as part of the Science Pipelines single-frame-processing (SFP) framework.
+A small number of additional values are also required, but can be quickly derived from the SFP results.
+The calculated values by Rapid Analysis are not to produce data products that are fundamental to continuing observations (`FAFF-REQ-0053`_), however, it is expected that observatory functionality will be reduced if an outage were to occur.
+This implies that the Rapid Analysis payload is not required to run at the summit, although it would be preferable to do so.
```

Comment:
"payload" or "workload" ? I think you are talking about "workload"...

Reply:
we've opted to use the word "payload" to signify that a pipeline framework (e.g. Prompt Processing) does the setup and handling of a pipeline execution, but the pipeline itself is the payload to the Prompt Processing framework (in this example).